pages/user.js

```diff
@@ -49,6 +119,7 @@ class User extends Component {
                                 src={isLoggedIn ? picture : ''}
                             />
                         </div>
+                        <h1>{hello}</h1>
```

Comment:
Again what is the use of this `hello` variable?

Reply:
I was using it initially for debugging